docs/source/reference/series.rst

```diff
@@ -30,6 +30,7 @@ Attributes
    Series.shape
    Series.size
    Series.empty
+   Series.T
```

Comment:
Series.transpose seems not existing in https://pandas.pydata.org/pandas-docs/stable/reference/series.html